frontend/public/components/utils/log-window.jsx

```diff
@@ -144,10 +146,14 @@ export class LogWindow extends React.PureComponent {
   }
 }
 
-LogWindow.propTypes = {
-  bufferFull: PropTypes.bool.isRequired,
-  lines: PropTypes.array.isRequired,
-  linesBehind: PropTypes.number.isRequired,
-  status: PropTypes.string.isRequired,
-  updateStatus: PropTypes.func.isRequired,
-};
+export const LogWindow = withTranslation()(LogWindowWithTranslation);
+
+LogWindow.propTypes =
+  WithTranslation &
```

Comment:
Is `WithTranslation` mean to use with `propTypes`? I thought that was for TypeScript types.